src/World.cpp

```diff
@@ -3477,9 +3602,13 @@ unsigned int cWorld::GetNumPlayers(void)
 
 
 
-int cWorld::GetNumChunks(void) const
+size_t cWorld::GetNumChunks(void) const
 {
-	return m_ChunkMap->GetNumChunks();
+	if (m_ChunkMap.get() != nullptr)
```

Comment:
I *think* this happened during world destruction, but #3176 might have eliminated this?
Let me investigate.